Makefile

```diff
@@ -79,6 +79,10 @@ endif
 	# Diff of uncommitted changes for running locally
 	git diff HEAD -- rasa | poetry run flake8 --select D --diff
 
+lint-security:
```

Comment:
My reasoning for running it separately is that we'd want to run it in another workflow with other security checks (cf. the GH workflow file). But if we want to run it with the rest of the CI checks, we could merge it with `make lint`.

I'd be cautious about merging them though because:
- if we end up merging checks, we do not achieve parallelism
- it'd make it hard for developers to see all the failing checks at once and they'd have to wait for other CI runs to see all errors (for instance: python lint + security lint both fail. You fix first the python lint issue, push your changes, **wait for another run**, and then realise the security lint fails too, etc...)

Reply:
Yup definitely agree with having them split in the CI. We could have an overriding `make` command for running all of them locally though. Like `make static-checks` or something.

Reply:
Ah yeah that's true, so it'd be only a developer tool 🤔

Reply:
Yeah - I find it useful to have one command I can run to ensure that I won't break any of the non-test checks in the CI.

Reply:
added ✅